modules/ROOT/pages/evaluate-dw-expressions.adoc

```diff
@@ -0,0 +1,57 @@
+= DataWeave Expression Watches
```

Comment:
Are these really called "watches," as the noun, and "expression" as an adjective: an expression watch?
Maybe we want...:
```suggestion
= Set DataWeave Expression Watches
```
OR
```suggestion
= Enable Watches for DataWeave Expressions
```

Reply:
I won't make any more edits to reword for "watch" being the adjective or verb rather than the noun, but please evaluate it based on how your audience will understand it best.

Reply:
"Debug Your App Using DataWeave Expressions"?